portfolio/src/main/webapp/script.js

```diff
@@ -31,3 +31,55 @@ function addRandomGreeting() {
   greetingContainer.innerText = greeting;
 }
 
+window.onload = function () {
+    // document.getElementsById('all').checked=true;
+    filterSelection('all') // Execute the function and show all columns
```

Comment:
writing comments is a dark art, where you want to avoid both over commenting and under commenting. A good rule of thumb, is to consider your code as if someone else was reading it, and then ask yourself, "could a programmer a little below my level understand what's happening here?". if the answer is no, first see if you can make your code more clear. if that doesn't help, then consider a comment.

to apply that to the comment here:
"execute the function" - it should be obvious to another programmer that this is executing a function, so this part of the comment is not helpful
"show all columns" - is this obvious? perhaps... but i would say it could be made more obvious by renaming the argument of filterSelection(...). "c" is not a descriptive name below, but "columns" or something like that might be. if you rename this to columns, then a reader can look at filterSelection, and it becomes obvious this is filtering comments - and then they have understood everything that's in the comment, and it becomes no longer necessary.

Reply:
Thanks Soonil. I made changes to the function and variable names. Sometimes when I bootstrap code, I forget to edit it properly to suit my Project. Editing helped me have a better understanding of what's really going on.